Cargo.toml

```diff
@@ -3,27 +3,28 @@ name = "gerritbot"
 version = "0.7.0"
 authors = ["boxdot <d@zerovolt.org>"]
 license = "MIT/Apache-2.0"
+edition = "2018"
 
 [dependencies]
-docopt = "0.8"
+docopt = "1.0"
 futures = "0.1"
 hyper = "0.10"
 hyper-native-tls = "0.2"
 iron = "0.6"
-itertools = "0.7"
-lazy_static = "0.2"
-log = "0.3"
-lru_time_cache = "0.7"
+itertools = "0.8"
+lazy_static = "1.3"
+log = "0.4"
+lru_time_cache = "0.9"
 notify-rust = "3"
-regex = "0.2"
-rlua = "0.12"
+regex = "1.1"
+rlua = "0.16"
 router = "0.6"
 rusoto_core = "0.29"
 rusoto_sqs = "0.29"
 serde = "1.0"
 serde_derive = "1.0"
```

Comment:
If you want, you can actually even remove this one, and just use:
```
serde = { version = "1.0", features = ["derive"] }
```

Then there is not need for `macro_use` anymore, just `use::{Serialize, Deserialize}` is enough.